src/run/formlyNgModelAttrsManipulator.js

```diff
@@ -18,7 +18,7 @@ function addFormlyNgModelAttrsManipulator(formlyConfig, $interpolate) {
       return template;
     }
     el.innerHTML = template;
-    var modelNodes = el.querySelectorAll('[ng-model], [data-ng-model]');
+    var modelNodes = el.querySelectorAll('[ng-model]:not([formly-skip-ng-model-attrs-manipulator]):not(.ignored-thing), [data-ng-model]:not([formly-skip-ng-model-attrs-manipulator]):not(.ignored-thing)');
```

Comment:
I updated the test to make the `.ignored-thing` class totally random. The idea behind this feature is to be able to specify any selector you like as part of the `data.skipNgModelAttrsManipulator` and have that value used in the `:not(` selector (but only if it is a string).